source/_integrations/rmvtransport.markdown

```diff
@@ -34,6 +34,11 @@ timeout:
   required: false
   default: 10
   type: integer
+scan_interval:
```

Comment:
`scan_interval` is [available](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval) for all polling platforms.

We don't mention it everywhere because there is a default interval for every platform that shouldn't hammer the API and avoid the running out of API calls if there is a limit.